src/tls13.c

```diff
@@ -4860,8 +4862,18 @@ static int CheckPreSharedKeys(WOLFSSL* ssl, const byte* input, word32 helloSz,
         return ret;
 #endif
 
-    /* Hash the rest of the ClientHello. */
-    ret = HashRaw(ssl, input + helloSz - bindersLen, bindersLen);
+    if (*usingPSK) {
```

Comment:
This troubled me for a while this week. Please have a look if you agree with the proposed fix.

The bug happens on TLSv1.3 server side, TLS or QUIC, no matter.

Reply:
This seems reasonable to me. @SparkiDev I would really like your feedback on this, since it is your code here. @icing does this relate at all to the early data PR's that have been put up?

Reply:
I'm good with that.
I can see that a bug is fixed here.